todo_app/data/trello_items.py

```diff
@@ -1,28 +1,17 @@
 import requests, json, os
+from todo_app import app
 from todo_app.data.item_class import Item
 
 
 def trello_get_items():
     board_id = os.getenv('BOARD_ID')
     api_key = os.getenv('TRELLO_API_KEY')
     token = os.getenv('TRELLO_API_TOKEN')
-    url = 'https://api.trello.com/1/boards/{board_id}/lists'.format(board_id=board_id)
-    headers = {
-        "Accept": "*/*"
-    }
-
-    query = {

```

Comment:
I'll admit I prefer the separate query, it makes it clearer that we don't need to think too hard about things like URL encoding, but not an important point!